KeychainExample/App.js

```diff
@@ -69,6 +69,19 @@ export default class KeychainExample extends Component {
     }
   }
 
+  async deleteAll() {
```

Comment:
`deleteAll = () => {...}` and call it without creating an arrow function in render body `onPress={deleteAll}`